### PR TITLE
I3162 dpulvariable

### PR DESCRIPTION
**Pull request by leefaisonr**

### CODE REVIEW

File: roles/dpul/meta/main.yml [hunk 1]
```diff
@@ -16,3 +16,16 @@ galaxy_info:
...
+  - role: deploy_user
+  - role: passenger
+  - role: redis
+  - role: postgresql
+  - role: nodejs
+  - role: openjdk
+  - role: imagemagick
+  - role: rails_app
+  # TODO: THis role runs with the line below commented out.  We need the line below to work
+  # - {role: solr, when: postgresql_is_local == true}
```
Comment (hackartisan) [inline roles/dpul/meta/main.yml:29]: Do we need this line to work? Is that for tests?

Comment (kayiwa) [reply roles/dpul/meta/main.yml]: I think this line is an error. Is there any reason one would install postgresql locally for dpul?

Comment (hackartisan) [reply roles/dpul/meta/main.yml]: dpul connects to the solrcloud. I think this line can simply be deleted.

**Review by hackartisan — CHANGES_REQUESTED** [review]
I ran this on staging and it looks good, but I'd like to get https://github.com/pulibrary/princeton_ansible/pull/2894 merged first please!

**Comment by hackartisan** [comment]
(I'm happy to rebase this again once that merge is done)

**Review by hackartisan — APPROVED** [review]

**Pull request merged**